dart/lib/src/sentry.dart

```diff
@@ -45,30 +45,28 @@ class Sentry {
   }
 
   /// Reports an [event] to Sentry.io.
-  static Future<SentryId> captureEvent(SentryEvent event) async {
-    return currentHub.captureEvent(event);
+  static Future<SentryId> captureEvent(SentryEvent event,
+      {dynamic hint}) async {
```

Comment:
please add a trailing comma :)

Reply:
thx, looks like `dartfmt -n --set-exit-if-changed ./` is not clever enough